test/runtests.jl

```diff
@@ -63,14 +66,39 @@ end
     h1D = Field(Centre, hdata, grid1D)
     u1D = Field(Face, udata, grid1D)
     
-    @test typeof(h1D) <: Field1D{Centre}
-    @test typeof(u1D) <: Field1D{Face}
+    @test typeof(h1D) <: Field1D{Centre, Grid1D{Periodic}}
+    @test typeof(u1D) <: Field1D{Face, Grid1D{Periodic}}
 
     @test h1D.grid == grid1D
     @test u1D.grid == grid1D
 
-    @test h1D.data == hdata
-    @test u1D.data == udata
+    #@test h1D.data == hdata
+    #@test u1D.data == udata
```

Comment:
?

Reply:
Why comment these out? If they are not relevant delete, if they need to be modified then fix?

Reply:
We can delete them now because they are just comparing interior points. We already have a test to check all points (interior + halo).

Reply:
Great! Delete them then!

Reply:
Done